model/task_annotations/task_annotations.go

```diff
@@ -0,0 +1,31 @@
+package task_annotations
+
+import (
+	"encoding/json"
+	"time"
+)
+
+type TaskAnnotation struct {
+	Id string `bson:"_id" json:"id"`
+	// comment about the failure
+	Note string `bson:"note,omitempty" json:"note,omitempty"`
+	// links to tickets definitely related.
+	Issues []IssueLink `bson:"issues,omitempty" json:"issue,omitempty"`
+	// links to tickets possibly related
+	SuspectIssues []IssueLink `bson:"suspected_issues,omitempty" json:"suspected_issues,omitempty"`
```

Comment:
nit: can  this just be called SuspectedIssues? It does read better / it's close enough to the bson/json that they may as well match

Reply:
good point! I'll switch it.